src/GNDStk/Node/test/call.test.cpp

```diff
@@ -0,0 +1,124 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+using namespace njoy::GNDStk;
+
+
+// -----------------------------------------------------------------------------
+// Some child_ts
+// -----------------------------------------------------------------------------
+
+// temperature_t
+struct temperature_t {
+   // just the metadata, for this example
+   double value;
+   std::string unit;
+};
+
+inline void convert(const Node<> &n, temperature_t &temp)
+{
+   temp.value = n(meta::dvalue);
+   temp.unit  = n(meta::unit);
+}
+
+// isotope_t
+struct isotope_t {
+   std::string symbol;
+   int A;
+   const Node<> *nuclides;
+};
+
+inline void convert(const Node<> &n, isotope_t &iso)
+{
+   iso.symbol = n(meta::symbol);
+   iso.A = n(meta::A);
+   iso.nuclides = &n(child::nuclides);
+}
+
+
+
+// -----------------------------------------------------------------------------
+// Scenario
+// -----------------------------------------------------------------------------
+
+SCENARIO("Testing GNDStk Node operator()") {
+
+   // If I put these outside SCENARIO, the build system claims that they're
```

Comment:
It's all about C++ scope. All the `.cpp` files in a directory are compiled together, which isn't unusual at all. Catch provides a `main()` for us. You can use `extern` to tell the compiler that it is defined in another file. 

Generally, when I want a variable/function available to all the tests in a directory, I put that variable in the (in this instance) `Node.test.cpp` file.

Reply:
In any event, we need to remove comments such as this before we tell others that they should use our library. While I understand your frustration, we don't necessarily want to air our dirty laundry for the rest of the world to see.

Reply:
Yes, thanks for noticing this, I'd forgotten about this comment. It'll be removed in my next upload. I guess I'm somewhat of a minimalist with regards to build and test systems, as I've found that they often put things together (or add stuff, like the main()) in ways that are non-obvious enough that using them is, in my experience at least, more confusing than just using good old-fashioned, tried and true language constructs. Like, in my WPEC talk, I used plain old C/C++ assert()s - which have been around for decades, and every viewer who has worked with C or C++ presumably knew exactly what they meant. :-)